capacity4more/modules/c4m/vocab/c4m_vocab_widget/c4m_vocab_widget.module

```diff
@@ -167,7 +167,8 @@ function c4m_vocab_widget_prepare_taxonomy_tree($terms) {
     // If the term is array, this vocab is of type "node".
     if(is_array($term)) {
       foreach($term as $id => $label) {
-        $options[$id]['label'] = check_plain($label);
+        // Decode ampersand back to & after sanitization.
+        $options[$id]['label'] = str_replace("&amp;", "&", check_plain($label));
```

Comment:
@YuriGizra 
This is not the correct solution: names with accents (Bélizè, Français, …) will still be broken…

Could it be that the labels are already run trough check_plain as the string is shown as `&amp;amp`in the list?